content/documentation/_index.md

```diff
@@ -31,15 +32,15 @@ Welcome to the IPFS Cluster documentation. This is where you will find user docu
 
 ## User guides
 
-* [Quickstart: Starting, enlarging and shrinking a cluster](/guides/quickstart)
+* [Quickstart: Starting, enlarging and shrinking a cluster](quickstart)
 
 ## Developer documentation
 
-* [Contribution guidelines](/developer/contribute)
-* [Project Architecture's overview](/developer/architecture)
+* [Contribution guidelines](developer/contribute)
```

Comment:
I don't feel 100% sure that changing this to relative urls won't break some things somewhere with hugo. Unless there is a major reason, we should keep this as is.

Reply:
The Hugo native nav won't work without changing the paths. I believe I set up redirects, though -- so the old paths should still work. Is there a place you know that these are linked *from* that I can test?

Reply:
Eh, nevermind. I can test that myself, and I'll make sure the redirects work. If the redirects work, do you have further concerns about changing the urls?

Reply:
Ah shoot. These comments should have been on the other note. Apologies. So....two points, in summary of *both* comments you left:

1) All of the url changes should have redirects and so nothing should break in the wild.

2) The Hugo standard is to *not* have the leading slash. Once I fixed other things in our file structure, the leading slash broke the paths. These need to be removed.